ui/helpers/utils/accounts.js

```diff
@@ -11,6 +11,7 @@ import { HardwareKeyringNames } from '../../../shared/constants/hardware-wallets
 // TODO: Remove restricted import
 // eslint-disable-next-line import/no-restricted-paths
 import { t } from '../../../app/scripts/translate';
+import { isSnapPreinstalled } from '../../../shared/lib/accounts';
```

Comment:
This one is wrong, it probably should be:
```suggestion
import { isSnapPreinstalled } from '../../../../shared/lib/snaps/snaps';;
```